hikari/api/rest.py

```diff
@@ -6897,6 +6903,7 @@ async def create_context_menu_command(
         name: str,
         *,
         guild: undefined.UndefinedOr[snowflakes.SnowflakeishOr[guilds.PartialGuild]] = undefined.UNDEFINED,
+        name_localizations: undefined.UndefinedOr[typing.Mapping[str, str]] = undefined.UNDEFINED,
```

Comment:
We also want to allow using the enum in all of these

Reply:
So should it be `typing.Mapping[locales.Locale, str]`? or something like `typing.Mapping[typing.Union[locales.Locale, str], str]`

Reply:
The union is what I would go for